src/cmdstan-guide/installation.Rmd

```diff
@@ -8,25 +8,30 @@ will install a pre-built version of CmdStan along with the required
 dependencies (i.e. a C++ compiler, a version of Make, and required
 libraries) detailed below under [Source installation].
 
-One can create a new conda environment and install CmdStan with
-the following command.
-
+One can create a new conda environment (named `stan`)
+and install CmdStan with the following command.
 ```
-conda create -n cmdstan -c conda-forge cmdstan
+conda create -n stan -c conda-forge cmdstan
 ```
 or install it in an existing environment with
 ```
 conda install -c conda-forge cmdstan
 ```
 
-With the conda installation, one can use the R or Python
-bindings to CmdStan, or can use the command `cmdstan_model`
-to activate the CmdStan makefile from anywhere. To enable either
-of these, ensure your environment is activated (via
-`conda activate cmdstan`, or whichever name you used above).
-
-Issues with the conda installation are handled separately.
-Please report conda-specific problems
+The conda installation is designed so one can use the R or Python
+bindings to CmdStan seamlessly. Additionally, it provides the command
+`cmdstan_model` to activate the CmdStan makefile from anywhere.
+To enable either of these, ensure your environment is activated^[A special
+environment, called `base`, is activated automatically by conda on startup.
+Installing software directly in `base` is not recommended by conda, but may be
+appropriate for some users who do not intend to use multiple environments]
+with `conda activate <env>`, where `<env>` is the name of
+the environment CmdStan was installed into. In the first command
+above, the environment was named `stan`, but any name may be used
+so long as it you are consistent.
```

Comment:
as it you -> as you

Also, if `stan` is changed, the name here will have to change.